### PR TITLE
feat: handle new attributes

### PR DESCRIPTION
**Pull request by CalvinRodo**
Handles the following global attributes: 
 -  manufacturer
 - android release version
 
Buckets the following attributes 
 - count on background-check
 - duration on background-process

Changes alarms on duration to p99 from average

### CODE REVIEW

File: server/aws/lambda/aggregate_values.js [hunk 1]
```diff
@@ -78,31 +149,59 @@ function pinDate(timestamp) {
...
+                            androidreleaseversion: androidreleaseversion
+                        };
+
+                    }
+
+                });
+            }
+
+        } catch(err) {
+            console.err(`issue parsing event: ${err}`);
```
Comment (obrien-j) [inline server/aws/lambda/aggregate_values.js:202]: I feel like you're trolling me here. :)

console.error() oui?

File: server/aws/lambda/aggregate_values.js [hunk 2]
```diff
@@ -137,15 +236,27 @@ function sendToDeadLetterQueue(payload, err) {
...
+                    console.error(`Failed updating sending to Dead Letter Queue ${err}`);
+                    sendToDeadLetterQueue(payload, err);
+                }
+            });
+        }
+
+    } catch(err){
+
+        //TODO: send to s3 bucket
+        console.error(`failed event: ${JSON.stringify(err)}`);
```
Comment (obrien-j) [inline server/aws/lambda/aggregate_values.js:256]: Not knowing what you are getting back here, but noting inconsistency with usage of err in the generatePayload exception handler.

**Review by obrien-j — APPROVED** [review]
LGTM.

File: server/aws/lambda/aggregate_values.js (hunk 1 above)
Comment (CalvinRodo) [reply server/aws/lambda/aggregate_values.js]: Fixed

File: server/aws/lambda/aggregate_values.js (hunk 2 above)
Comment (CalvinRodo) [reply server/aws/lambda/aggregate_values.js:256]: Also removed so we aren't logging out things we shouldn't be.

**Pull request merged**